src/System.IdentityModel.Tokens.Jwt/ClaimTypeMapping.cs

```diff
@@ -96,6 +99,7 @@ internal static class ClaimTypeMapping
             { "deviceregid", "http://schemas.microsoft.com/2012/01/devicecontext/claims/registrationid" },
             { "endpointpath", "http://schemas.microsoft.com/2012/01/requestcontext/claims/x-ms-endpoint-absolute-path" },
             { "forwardedclientip", "http://schemas.microsoft.com/2012/01/requestcontext/claims/x-ms-forwarded-client-ip" },
+            { "fwt", "http://schemas.microsoft.com/2012/01/requestcontext/claims/x-ms-forwarded-client-ip" },
```

Comment:
Why add another one?

Reply:
We can remove it. I thought this claim (IP Address) was the same as forwardedclientip.
BTW, this is generated code. This one was added because the third boolean on this line: https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/blob/2b89eeae784e16eb8434aec16e9ad7d6c6b215c3/src/System.IdentityModel.Tokens.Jwt/ClaimsKnowledge.tti#L114 is set to true. I'll set it to false.

Reply:
Removed: https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/commit/bb86cda55ee203f838e5062fcd781c60fdedfffa